api/src/pcapi/routes/backoffice/templates/layouts/connected.html

```diff
@@ -78,10 +78,10 @@ <h6 class="mdc-deprecated-list-group__subheader">{{ name }}</h6>
                 {% call menu_section("Acteurs culturels") %}
                   {% if has_permission("READ_PRO_ENTITY") %}
                     {{ menu_section_item('Liste des acteurs culturels', 'backoffice_web.pro.search_pro') }}
-                    {{ menu_section_item('Structures à valider', 'backoffice_web.validation.list_offerers_to_validate') }}
+                    {{ menu_section_item('Entités juridiques à valider', 'backoffice_web.validation.list_offerers_to_validate') }}
                     {{ menu_section_item('Rattachements à valider', 'backoffice_web.validation.list_offerers_attachments_to_validate') }}
                   {% endif %}
-                  {% if has_permission("READ_TAGS") %}{{ menu_section_item('Tags structure', 'backoffice_web.offerer_tag.list_offerer_tags') }}{% endif %}
+                  {% if has_permission("READ_TAGS") %}{{ menu_section_item('Tags entité juridique', 'backoffice_web.offerer_tag.list_offerer_tags') }}{% endif %}
```

Comment:
Détail : On peut écrire :
```suggestion
                  {% if has_permission("READ_TAGS") %}{{ menu_section_item('Tags des entités juridiques', 'backoffice_web.offerer_tag.list_offerer_tags') }}{% endif %}
```
C'est plus joli et raccord avec `'Tags des offres et partenaires culturels'` juste en-dessous